modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -64,6 +60,10 @@ endif::[]
 ifeval::["{context}" == "installing-vmc-user-infra"]
 :vmc:
 endif::[]
+ifeval::["{context}" == "installing-restricted-networks-vsphere"]
+:vsphere:
+:restricted:
+endif::[]
```

Comment:
Can you clarify what is happening in this module by moving these ifeval statements? Thanks!

Reply:
This likely stemmed from a merge conflict resolution. I'll revert this since the change has no purpose.